student-work/2023/Sharon-Schnee/api-final-project/post.md

```diff
@@ -1 +1,112 @@
+pattyWeightGr | int  | N | The patty weight in grams. Options include 100, 200, and 300. | 100 |
+pattyCook | string | N |  How the patty is cooked. Options include rare, mediumRare, mediumWell, and wellDone.| mediumWell|
+bunType | string  | N | The type of bun, or none, for no bun. Bun types include white, wholeWheat, sourdough, and glutenFree. | white | 
+condiment1 | string | N |  First choice of condiment. Options include ketchup, bbqSauce, garlicMayonnaise, mayonnaise, mustard, honeyMustard, thousandIsland, ranchDressing, and none | ketchup |
+condiment2 | string | N |  Second choice of condiment. Options include ketchup, bbqSauce, garlicMayonnaise, mayonnaise, mustard, honeyMustard, thousandIsland, ranchDressing, and none | none |
+topping1 | string  | N | First choice of topping. Options include pickles, onion, lettuce, tomato, jalapeno, sauteedOnions, relish, and none | none |
+topping2 | string  | N | Second choice of topping. Options include pickles, onion, lettuce, tomato, jalapeno, sauteedOnions, relish, and none | none |
+topping3 | string  | N | Third choice of topping. Options include pickles, onion, lettuce, tomato, jalapeno, sauteedOnions, relish, and none | none |
+extraTopping | string | N | Choice of extra topping. Options include cornBeef, pastrami, pulledeeBf, oneEgg, and twoEggs | none |
+
+#### Side object
+Name | type | Required | Description | Default 
+---- | ----- | ----- | -------------------- | ---
+type | string | N |  Choice of side. Options include frenchFries, onionRings, sweetPotatoFries, spicyFries, coleSlaw, sideSalad, and none. | frenchFries |
+size | string | N |Size of side order. Options include medium and large. | medium |
+
+#### Drink object
+Name | type | Required | Description | Default 
+---- | ----- | ----- | -------------------- | ---
+type | string  | N | Type of drink. Options include coke, dietCoke, sprite, iceTea, orangeJuice, appleJuice, mangoNectar, peachNectar, and none | coke |
+size | string | N |  Size of drink. Options include small, medium, large, and xLarge. | medium |
+drink | boolean  | N | With or without ice. Options include true=with ice, false=without ice. | true |
+
+<!-- Replace the two example rows and include rows for all your parameters. -->
+<!-- If one of the parameters has a set of sub-parameters, create a table or bulleted list for that, but proceed with caution. If the API is complex, there might be an easier way to do your reference section than writing markup by hand. -->
+
+## Examples
+
+### Request
+
+Include the authorization header using the following structure. For more information about Authentication, see the [ReadMe](https://github.com/Schnee18/student-showcase/blob/main/student-work/2023/Sharon-Schnee/api-final-project/readme.md) file.
+
+```
+curl --request GET https://api.gphd.net/lunch/burgermeal
+--header 'Content-Type: application/json'
+--header 'Authorization: Bearer ImahFWLZWFdD8VVcUtIED2YuOjPFlZpldQTE5tUqKdv'
```

Comment:
Lines 55 and 56 are examples for authorization (using HTTP headers) so if you want to include it in that section you can.